package.json

```diff
@@ -7,7 +7,7 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "build": "babel src --source-maps --out-dir lib",
+    "build": "babel src --source-maps --out-dir lib && cp ./src/ansi_to_html.d.ts ./lib",
```

Comment:
I am not that familiar with babel; but won't the `--copy-files` argument suit the needs here? [https://babeljs.io/docs/en/babel-cli#copy-files](url) In my clone of master this appears to work fine, and removes the need of adding an extra call with hardcoded paths.